wideq/client.py

```diff
@@ -432,7 +432,15 @@ def _get_config(self, key):
             self.device.id,
             key,
         )
-        return json.loads(base64.b64decode(data).decode('utf8'))
+        data = base64.b64decode(data).decode('utf8')
+        try:
+            return json.loads(data)
+        except json.decoder.JSONDecodeError:
+            LOGGER.warning(
+                'JSONDecodeError malformed json (%s)', data)
+            # Added fix to correct malformed JSON!
+            # see https://github.com/sampsyo/wideq/issues/64
+            return json.loads(data.replace('{[', '[').replace(']}', ']'))
```

Comment:
Maybe we should catch malformed JSON in the _second_ call too and raise a more useful APIError?

Reply:
ok I add another new APIError into the core